app/controllers/application_controller/filter.rb

```diff
@@ -1,10 +1,13 @@
 # Filter/search/expression methods included in application.rb
+require_relative '../../helpers/miq_expression/filter_subst'
```

Comment:
@Fryguy 
These methods do the substitution into an expression given a set of input parameters. Till now that feature was UI specific, since UI was the only place you could supply user inputs. Maybe @gtanzillo @dclarizio might have more ideas on it.